action.yml

```diff
@@ -32,13 +32,14 @@ inputs:
     description: Verbose mode enabled
     required: false
     default: "true"
+  region:
+    description: The region where to send/read the e2e data
```

Comment:
Could you add the possible values here , I think they are `US`, `EU` and `Staging` .